interactive/templates/emails/welcome_email.html

```diff
@@ -0,0 +1,15 @@
+{% autoescape off %}
+<p>Hi {{ name }},</p>
+<p>Thank you for your interest in OpenSAFELY Interactive.</p>
+<p>
+  Please click on the link to finish setting up your account<br />
+  {% block reset_link %}
+      <a href="{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}">
+        {{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
```

Comment:
You probably want to generate the full URL and pass that into the context.  It avoids accidentally broken links if you use something like `furl` to smush the domain and path together and generally makes debugging easier when your `reverse()` blows up in a python module rather than inside the template renderer!

Reply:
That's a good point, although I think I copied that straight from the Django source code! I'll write a separate issue for that and maybe take a look at it later on :)

Reply:
I think this is probably a common pattern, but I've had to debug its failure states in the past so many times that This One Neat Trick™ (etc) makes it much easier (it's inevitably a double slash between the domain and the path too 😩) to get to the bottom of (eg you can usually see the issue in a Sentry traceback).